src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure implements Drawable, AreaCalculable {
+    private double side;
+
+    public Square(double side, String color) {
+        super.setColor(color);
+        this.side = side;
+    }
+
+    @Override
+    public double area() {
+        return Math.pow(side,2);
```

Comment:
All magic numbers in your code should be constants. Consider defining '2' as a constant with a clear name that indicates its purpose.